.github/ISSUE_TEMPLATE.md

```diff
@@ -49,6 +71,19 @@ What currently happens
 
 What you would like to happen
 
-### Use case
+### Usecase
 
 Please provide use cases for changing the current behavior
+
+### Additional information
+
+- Is this request for your company? Y/N
+
+..- If Y: Company name:
```

Comment:
Same comment as in other projects. If this is intended to be a sub-bullet please use two spaces before the `-`.

Reply:
Fixed, thanks